setup.py

```diff
@@ -128,6 +128,7 @@
         "cds_ils.migrator.models": [
             "cds_book = cds_ils.importer.providers.cds.models.book:model",
             "cds_standard = cds_ils.importer.providers.cds.models.standard:model",
+            "ils_journal = cds_ils.importer.providers.cds.models.journal:model",
```

Comment:
this entrypoint should be in series_models group
I see there is a bug, please rename below (and all the ccurences): 
`cds_ils.importer.series_models` -> ` cds_ils.migrator.series_models` and move the journals model there